rs/tests/driver/src/driver/bootstrap.rs

```diff
@@ -597,32 +580,17 @@ pub fn configure_setupos_image(
     name: &str,
     nns_url: &Url,
     nns_public_key: &str,
-    to: &NestedVersionTarget,
 ) -> anyhow::Result<PathBuf> {
     let tmp_dir = env.get_path("setupos");
     fs::create_dir_all(&tmp_dir)?;
 
-    let setupos_image = match to {
-        NestedVersionTarget::Branch(false) => {
-            env.get_dependency_path("ic-os/setupos/envs/dev/disk-img.tar.zst")
-        }
-        NestedVersionTarget::Branch(true) => {
-            bail!("Starting nested VMs from `-test` versions is unsupported.");
-        }
-        NestedVersionTarget::Mainnet => {
-            let url = env.get_mainnet_setupos_img_url()?;
-
-            download_setupos_image(&tmp_dir, &url, None)?
-        }
-        NestedVersionTarget::Published {
-            version: _,
-            url,
-            sha256,
-        } => download_setupos_image(&tmp_dir, url, Some(sha256.to_owned()))?,
-    };
+    let setupos_image = PathBuf::from(std::env::var(SETUPOS_PATH_ENV_VAR).unwrap_or_else(|_| {
+        eprintln!("environment variable {SETUPOS_PATH_ENV_VAR} is not set");
+        process::exit(1);
```

Comment:
I'd avoid process::exit because it terminates the process without running the destructors. Consider instead returning the error via Result or panicing which both run destructors.

Reply:
I swear I pulled this from elsewhere in the driver, but it is by far in the minority, fixed.